### PR TITLE
ETFE-996: Added feature switching functionality

### PR DESCRIPTION
**Pull request by r-melvin**

### CODE REVIEW

**Review by MJCallahanPage — CHANGES_REQUESTED** [review]
Just a couple of queries

File: conf/application.conf [hunk 1]
```diff
@@ -62,6 +63,16 @@ microservice {
...
+      }
+       emcs-tfe-report-a-receipt-frontend {
+        protocol = http
+        host = localhost
+        port = 8313
+      }
+      emcs-tfe-create-movement {
+        protocol = http
+        host = localhost
+        port = 8314
```
[the remark this answers is not in the document]
Comment (r-melvin) [reply conf/application.conf:75]: This is needed so the feature switching module can refresh and reload the changes once any have been made

File: conf/application.conf [hunk 2]
```diff
@@ -62,6 +63,16 @@ microservice {
         protocol = http
         host = localhost
         port = 8311
+      }
+       emcs-tfe-report-a-receipt-frontend {
+        protocol = http
+        host = localhost
+        port = 8313
+      }
```
[the remark this answers is not in the document]
Comment (r-melvin) [reply conf/application.conf:71]: We don't need this right now but the module allows us to have all the feature switches for the various services on one page to make testing and adjustments easier. When the feature switches are implemented in RoR, the page would pick them up like so
Screenshot 2023-05-24 at 15 02 10

File: conf/application.conf [hunk 3]
```diff
@@ -17,7 +17,7 @@ include "frontend.conf"
 appName="emcs-tfe-create-movement-frontend"
 deskproName="emcstfe"
 
-play.http.router = testOnlyDoNotUseInAppConf.Routes
+play.http.router = prod.Routes
```
[the remark this answers is not in the document]
Comment (r-melvin) [reply conf/application.conf:20]: Sure, I can change this

**Review by MJCallahanPage — APPROVED** [review]

**Pull request merged**